lib/charms/operator_libs_linux/v1/snap.py

```diff
@@ -236,7 +238,28 @@ def _snap(self, command: str, optargs: Optional[Iterable[str]] = None) -> str:
         try:
             return subprocess.check_output(_cmd, universal_newlines=True)
         except CalledProcessError as e:
-            raise SnapError("Could not %s snap [%s]: %s", _cmd, self._name, e.output)
+            raise SnapError("Could not {} for snap [{}]: {}".format(_cmd, self._name, e.output))
+
+    def _snap_daemons(
```

Comment:
I don't like the name. It reads like a getter. Add a verb for clarity: _snap_daemons_run?

Also why is it daemons plural? _snap_daemon_run?

Also why is there a daemon in there? is it interesting that it's a daemon? _snap_run?

Reply:
It was an attempt to leave a low footprint, as it's mostly the same command as `Snap._snap()` but for `daemons`.

I can rename the former to `Snap._snap_run()` and `Snap._snap_daemon_run()`, what do you think?

Reply:
Aha no then I'd say uniformity with existing code takes precedence, we have _snap, and this apparently is a daemon thing, so let's keep `_snap_daemon`. 

(I don't know much about snaps)